src/molecules/Toggle/index.jsx

```diff
@@ -0,0 +1,45 @@
+export class Toggle extends React.Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+      isActive: false
+    };
+    this.toggleSwitch = this.toggleSwitch.bind(this);
+  };
+
+  componentDidMount(){
+    const {
+      onChange
+    } = this.props
+  };
+
+  toggleSwitch(){
+    this.state.isActive ? this.setState({ isActive: false }) : this.setState({ isActive: true});
```

Comment:
You could make this a bit more graceful: 
```js
this.state.isActive = !this.state.isActive;
```

However, if that logic isn't clear enough we would want to drop a comment, or keep your version which (while more verbose) gives more context to what's happening.